src/v35.js

```diff
@@ -16,7 +16,7 @@ function stringToBytes(str) {
 export const DNS = '6ba7b810-9dad-11d1-80b4-00c04fd430c8';
 export const URL = '6ba7b811-9dad-11d1-80b4-00c04fd430c8';
 
-export default function (name, version, hashfunc) {
+function v35(name, version, hashfunc) {
```

Comment:
Why not just `export default function v35(…`?

Reply:
I actually prefer this, I just went with matching what was in src/v4.js 👍